src/fontra/client/lang/en.json

```
@@ -60,6 +73,7 @@
   "editor.pen-tool": "Pen Tool",
   "editor.pointer-tool": "Pointer Tool",
   "editor.power-ruler-tool": "Power Ruler Tool",
+  "editor.shape-tool": "Shape Tool",
   "editor.shape-tool-ellipse": "Shape Tool Ellipsis",
```

This typo must have existed for a while...
```suggestion
  "editor.shape-tool-ellipse": "Shape Tool Ellipse",
```